lms/templates/discovery/filter.underscore

```diff
@@ -0,0 +1,4 @@
+  <a data-value="<%- query %>" class="facet-option" href="#" data-type="<%- type %>">
```

Comment:
Whenever I see an ```<a href="#">``` it usually means that a ```<button>``` should be used instead.  Buttons do something and links go somewhere.   I think this qualifies as doing something, so it should be a button.  We should also consider what happens to the focus when the button is invoked.  Does it jump to search results or stay where it is.  IF the user is likely to choose multiple filters, then I would say focus should stay on the button.  Whatever happens, it shouldn't move to the top of the page (which will happen if page actually reloads)